<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ad_add_Dialog</class>
 <widget class="QDialog" name="ad_add_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加账号</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>211</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>用户类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>手机号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>设置密码：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>顾客</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>服务员</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>厨师</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>经理</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>系统管理员</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pnlineEdit">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pwdlineEdit">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>40</number>
      </property>
      <item>
       <widget class="QPushButton" name="add_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
